frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -263,6 +285,24 @@ const TAGGED_TEMPLATES = {
       </>
     );
   },
+  enableMacDiskEncryption: (activity: IActivity) => {
+    return (
+      <>
+        {" "}
+        enforced disk encryption for macOS hosts{" "}
+        {getDiskEncryptionMessageSuffix(activity.details?.team_name)} .
```

Comment:
Is there an extra space before the period?

Reply:
@gillespi314 all of these (spaces pre-period, line break) were auto-formatted. Something to do with the way JSX whitespace is parsed? Seems like the only real spaces or breaks are these ones: `{" "}` . In the end they all render as they should though.